python_modules/dagster/dagster_tests/core_tests/definitions_tests/test_partitioned_schedule.py

```diff
@@ -45,10 +45,8 @@ def test_daily_schedule():
     def my_partitioned_config(start, end):
         return {"start": str(start), "end": str(end)}
 
-    assert my_partitioned_config(datetime(2021, 5, 7), datetime(2021, 5, 8)) == {
```

Comment:
this just did a direct invocation of the decorated function, so it would return whatever you passed it, whether it aligned with the partition or not